tests/TestResponse.py

```diff
@@ -1,18 +1,23 @@
-import pytest 
-import time
-from http.server import BaseHTTPRequestHandler, HTTPServer
-from testServer import MyServer
+from drops.DropsDriver import myClient
+from testServerSpawner import ServerSpawner
+from drops.helpers.JsonFileHandler import JsonFileHandler
+import json
 
+ip = "127.0.0.1"
+port = 8081
+supported_json = "drops/supported.json"
 
 class TestResponse:
-  def test_responses(self, capsys):
-    hostName = "localhost"
-    serverPort = 8081
-    supportedJson = '../supported.json'
+    def test_get_status(self, capsys):
```

Comment:
part of what makes unit tests so awesome is they communicate intent/functionality to other developers. 

What you have here is clean and sweet and shows the intentionality of this design. Would be made even more potent with comments explaining what each step does.